packages/react-native-editor/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "gutenberg-mobile",
+  "name": "@wordpress/react-native-editor",
```

Comment:
It's marked as private, it might be concerning when publishing to npm. I assume that it's going to be referenced as an npm dependency from now on for packages that are published.

Reply:
Looking at #18515, it feels like we might keep those packages private for some time as long as they are used only in `*.native.js` files and not included in `package.json` files of other packages.

How about we make all 3 packages updated in this PR private until we have the migration finished? :)

Reply:
I set them to private here - https://github.com/WordPress/gutenberg/pull/18531/commits/38bf6c61e27cf73592c76a2c7b2dbd47bd71dac2